src/common/com/intellij/plugins/haxe/util/HaxeResolveUtil.java

```diff
@@ -526,8 +526,13 @@ public static HaxeClass tryResolveClassByQName(@Nullable PsiElement type) {
       return null;
     }
 
+    // can be accessed without full QName
+    HaxeClass result = findClassByQName(type.getText(), type.getContext());
```

Comment:
https://github.com/eliasku/intellij-haxe/pull/2/files
@EBatTiVo Please look at, which way is better